### DISCUSSION TITLE
Good Ebiten v2 tutorial :)

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [hajimehoshi/ebiten#2041] [Q&A] [answered]
Hey there,

I want to start learning Ebiten to develop games :)
I'm still very noob, so I'd like to know what are the best tutorials/articles that you recommend reading/watching to get into the world of Ebiten? 😅 

(Maybe some "get started" tutorial for version 2)

**Comment by hajimehoshi** [comment, marked as answer]
Hi,

What about these tutorials?

* https://www.fatoldyeti.com/categories/roguelike-tutorial/
* https://www.joshdeprez.com/77
* https://golangprojectstructure.com/creating-cool-games-with-ebiten-in-go/

**Reply by 201flaviosilva** [reply]
Nice, that's it, I'll check it out, thank you very much, and sorry for the noob question 😅 (Sometimes I should do better research)

**Reply by yuuuuuuan** [reply]
there are what i need!!!thanks a lot
